formulas/github/functions/github-get-archive

```diff
@@ -33,5 +41,5 @@ githubGetArchive() {
         args[${#args[@]}]="$destination"
     fi
 
-    curl ${args[@]} "https://github.com/$repository/tarball/$tag"
+    curl "${args[@]}" "https://github.com/$repository/tarball/$tag"
```

Comment:
:-(  I wish this used `wickGetUrl`.  Any chance you want to add `--username` and `--password` options to that function so that this could use either `curl` or `wget`, depending on the system?